README.md

```diff
@@ -573,6 +573,11 @@ function render () {
   console.log(`${post.title} for ${profile.name}`)
 }
 
+profile.subscribe(render)
```

Comment:
Calling `subscribe()` has no reason for this case.

Maybe we can explain the difference on some more reasonable example with a single store?

Reply:
> Maybe we can explain the difference on some more reasonable example with a single store?

I agree. That's better than an arbitrary example that can't be understood. What would that example be like?

TBH, It would be better if the doc could explain common methods like `subscribe`, `listen` etc.